src/tools/genpolicy/rules.rego

```diff
@@ -1319,6 +1319,35 @@ ExecProcessRequest {
     print("ExecProcessRequest 3: true")
 }
 
+UpdateInterfaceRequest {
+    print("UpdateInterfaceRequest 1: input =", input)
+
+    i_interface := input.interface
+    p_allowed_raw_flags := policy_data.request_defaults.UpdateInterfaceRequest.allow_raw_flags
+
+    print("UpdateInterfaceRequest 1: i_interface.raw_flags =", i_interface.raw_flags)
+    print("UpdateInterfaceRequest 1: p_allowed_raw_flags =", p_allowed_raw_flags)
+
+    # The agent only checks certain raw_flags; reject anything not processed.
+    bits.and(i_interface.raw_flags, bits.negate(p_allowed_raw_flags)) == 0
+
+    p_forbidden_names := policy_data.request_defaults.UpdateInterfaceRequest.forbidden_names
+
+    print("UpdateInterfaceRequest 1: i_interface.name =", i_interface.name)
```

Comment:
No need to print again the i_interface.name?

Maybe also print the entire contents of policy_data.request_defaults.UpdateInterfaceRequest using a single print(), instead of printing the flags and the name separately.